lib/models/fl_countries.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Joshua de Guzman (https://joshuadeguzman.github.io). All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import 'fl_country.dart';
+
+class FLCountries {
+  String status;
+  Result result;
+  String requestId;
```

Comment:
Hi @krtliv, this is the base payload of the response. Can you please just extract the `countries` result itself?

Please see the base payload here:
https://github.com/joshuadeguzman/freelancer-sdk-flutter/blob/master/lib/models/fl_response.dart

Reply:
hi @joshuadeguzman i removed FLCountries and added a List<FLCountry> countries inside FLProject, i also added the tests.